app/views/microposts/edit.html.erb

```diff
@@ -1,32 +1,8 @@
-<%= provide(:title, 'Activity-Edit-Page') %>
-<h3>Editing Micropost</h3>
+<h1>Editing Microposts</h1>
 
-<!--# render 'form', micropost: @micropost -->
-
-<%= form_with(model: @micropost) do |form| %>
-  <section>
-    <div>
-
-      <div>
-        <div>
-
-          <%= form.label :content %>
-          <%= form.text_area :content %>
-
-          <input type="hidden" name="id" value=<%=session[:user_id]%>>
-
-          <br>
-            <div class="actions">
-             &nbsp;&nbsp;<%= form.submit "Update my Blog" %>
-             <%= link_to 'Back', user_path(current_user), class: 'btn btn-secondary'%>
-
-            </div>
-        </div>
-      </div>
-
-    </div>
-  </section>
-<% end %>
+<%= render 'form', micropost: @micropost %>
```

Comment:
@HrisikeshSinghaChowdhury  why do we need `micropost: @micropost` ? where is it used?

Reply:
It's not needed here because we are not passing any parameter as such to form partials

Reply:
@Ramyani-QI As in the 'form partials we are directly attaching the form id with model microposts.....so we donnot need to pass and use any extra parameter during rendering.....
I may be wrong